### ISSUE TITLE
Docker image creation for 4.8.0 Alpha 2 is failing

### ISSUE THREAD

**Issue by juliamagan** [wazuh/wazuh-packages#2762]
|Wazuh version|Install type|Action performed|
|---|---|---|
| v4.8.0-alpha2 | Docker | Image genaration | 

## Description

During the creation of packages for 4.8.0 Alpha 2, carried out in https://github.com/wazuh/wazuh-jenkins/issues/6147, we get the following error and we are not able to create the images:

```
11:15:53  ------
11:15:53  #16 CANCELED
11:15:53  failed to solve: executor failed running [/bin/sh -c apt-get update &&     apt-get install wazuh-manager=${WAZUH_VERSION}-${WAZUH_TAG_REVISION}]: exit code: 100
11:15:53  [Pipeline] sh
11:15:53  + docker images
11:15:53  REPOSITORY   TAG       IMAGE ID   CREATED   SIZE
11:15:53  [Pipeline] }
11:15:53  [Pipeline] // stage
11:15:53  [Pipeline] stage
11:15:53  [Pipeline] { (Tag and Push Wazuh images)
11:15:53  [Pipeline] readProperties

11:15:54  [Pipeline] sh
11:15:54  + docker tag wazuh/wazuh-manager:4.8.0 wazuh/wazuh-manager:4.8.0-alpha2
11:15:54  Error response from daemon: No such image: wazuh/wazuh-manager:4.8.0
```

More info can be found in the following build: https://ci.wazuh.info/job/procedure_push_docker_images/120/console

**Comment by davidcr01** [comment]
# Update Report

## Reproducing the error

I reproduced the error in local by building the Docker images, using the `build-docker-images/build_images.sh` script. The error itself is not Docker-related, but Wazuh manager package installation:

```console
=> [wazuh/wazuh-dashboard:4.8.0 stage-1 12/12] WORKDIR /usr/share/wazuh-  0.2s
 => [wazuh/wazuh-dashboard:4.8.0] exporting to image                      21.2s
 => => exporting layers                                                   21.2s
------
 > [wazuh/wazuh-manager:4.8.0  7/20] RUN apt-get update &&     apt-get install wazuh-manager=4.8.0-1:
#0 1.111 Hit:1 http://archive.ubuntu.com/ubuntu jammy InRelease
#0 1.111 Hit:2 http://archive.ubuntu.com/ubuntu jammy-updates InRelease
#0 1.112 Hit:3 http://security.ubuntu.com/ubuntu jammy-security InRelease
#0 1.149 Hit:4 http://archive.ubuntu.com/ubuntu jammy-backports InRelease
#0 2.145 Get:5 https://packages-dev.wazuh.com/pre-release/apt unstable InRelease [17.3 kB]
#0 3.277 Get:6 https://packages-dev.wazuh.com/pre-release/apt unstable/main amd64 Packages [41.5 kB]
#0 3.435 Fetched 58.7 kB in 3s (19.1 kB/s)
#0 3.435 Reading package lists...
#0 4.362 Reading package lists...
#0 5.239 Building dependency tree...
#0 5.442 Reading state information...
#0 5.702 Suggested packages:
#0 5.702   expect
#0 5.733 The following NEW packages will be installed:
#0 5.733   wazuh-manager
#0 7.117 0 upgraded, 1 newly installed, 0 to remove and 2 not upgraded.
#0 7.117 Need to get 288 MB of archives.
#0 7.117 After this operation, 888 MB of additional disk space will be used.
#0 7.117 Get:1 https://packages-dev.wazuh.com/pre-release/apt unstable/main amd64 wazuh-manager amd64 4.8.0-1 [288 MB]
#0 269.0 debconf: delaying package configuration, since apt-utils is not installed
#0 269.1 Fetched 288 MB in 4min 23s (1096 kB/s)
#0 269.2 Selecting previously unselected package wazuh-manager.
(Reading database ... 5970 files and directories currently installed.)
#0 269.5 Preparing to unpack .../wazuh-manager_4.8.0-1_amd64.deb ...
#0 269.5 Unpacking wazuh-manager (4.8.0-1) ...
#0 287.8 Setting up wazuh-manager (4.8.0-1) ...
#0 288.8 tar (child): xz: Cannot exec: No such file or directory
#0 288.8 tar (child): Error is not recoverable: exiting now
#0 288.8 tar: Child returned status 2
#0 288.8 tar: Error is not recoverable: exiting now
#0 288.8 dpkg: error processing package wazuh-manager (--configure):
#0 288.8  installed wazuh-manager package post-installation script subprocess returned error exit status 2
#0 288.8 Errors were encountered while processing:
#0 288.8  wazuh-manager
#0 288.9 E: Sub-process /usr/bin/dpkg returned an error code (1)
------
failed to solve: process "/bin/sh -c apt-get update &&     apt-get install wazuh-manager=${WAZUH_VERSION}-${WAZUH_TAG_REVISION}" did not complete successfully: exit code: 100
```
It seems that the `tar` command is failing in the post-installation process of the Wazuh manager package.

@nico-stefani encountered that the `xz-utils` was included as a dependency in the Wazuh manager package, but this change is not included in 4.8.0 Alpha2: https://github.com/wazuh/wazuh-packages/issues/2731

**Comment by teddytpc1** [comment]
This bug will be resolved [here](https://github.com/wazuh/internal-devel-requests/issues/666).
